### ISSUE TITLE
error: option --no-svn-revision not recognized with 3.9/unreleased

### ISSUE THREAD

**Issue by zopyx** [Jarn/jarn.mkrelease#9]
I just tried the latest  version from master:

ZOPYX-MBP:zopyx.plone.smashdocs ajung$ bin/mkrelease  -d pypi -T
Releasing zopyx.plone.smashdocs 0.2.16
running egg_info
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --no-svn-revision not recognized
egg_info failed

**Comment by stefanholek** [comment]
Hm, you use some super-new setuptools?

**Comment by zopyx** [comment]
Not really

```
ZOPYX-MacBook-Pro:zopyx.smashdocs ajung$ bin/pip freeze
appdirs==1.4.0
click==6.7
furl==0.5.6
jarn.mkrelease==3.9
lazy==1.2
orderedmultidict==0.7.11
packaging==16.8
py==1.4.32
PyJWT==1.4.2
pyparsing==2.1.10
pytest==3.0.6
requests==2.13.0
setuptools-git==1.1
setuptools-hg==0.4
setuptools-subversion==3.1
six==1.10.0
-e git+git@github.com:zopyx/zopyx.smashdocs.git@d576a0745ab2fd4836dde257eb1ba3cdee3c78cc#egg=zopyx.smashdocs

```

**Comment by stefanholek** [comment]
Oh well, removed in setuptools v33:
http://setuptools.readthedocs.io/en/latest/history.html#v33-0-0

**Comment by zopyx** [comment]
But pinning setuptools=32.0.0 does not help..same error

**Comment by stefanholek** [comment]
Fixed in 3.10. Thanks for the report.
